docs/MODULES.md

```diff
@@ -194,6 +270,20 @@ or with IPv6 and subnet
 4567890123456789012345678901234567890123456789012345678901234567 10.10.0.4 fd00::3 fd00:1::/64
 ```
 
+##### IPTunnel Client and Interent Peer
+
+If you wish to run an IPTunnel Client on the same node as as an Internet Peer, you will need to create a route to that node over the internet connection available.
+
+The reason of doing this is to allow the Yggdrasil internet peer to not try to use the new path to the internet that it aquies by forming the tunnel to peer.  The peer conenction cannot shuttle tunnel connections of the same tunnel.
```

Comment:
This sentence is unclear, and I am not sure what it means. It should be reworked.

Reply:
it was 3 paragraphs before.

how about this

> If a route is not created to the peer, Yggdrasil peering will try to route over the tunnel.  Since the tunnel is depended on the peer the peer cannot function over the tunnel. Peer tries to feed packets over the tunnel, that tries to feed packets to the peer causing both to fail.

Reply:
> This must be done, because since all Internet traffic will be redirected over the tunnel and a route is not created to the peer, Yggdrasil peering will try to route over the tunnel. Since the tunnel depends on the peer, the peer will not function, which will collapse the tunnel. In other words, the peer tries to feed packets over the tunnel, that tries to feed packets to the peer, causing both to fail.

Does that work? I find it's a bit more clear, and I corrected some grammar.